### DISCUSSION TITLE
Partial salvage of wrecked subs

### DISCUSSION THREAD

**Discussion by WickedWolfy** [FakeFishGames/Barotrauma#7949] [Ideas and suggestions]
I am not asking for a "complete" salvage of wrecked subs - that would be unreasonable.

But at least some salvaging of metals and components, like copper or lead, should be possible.
I mean the downed sub is a giant hunk of steel! My salvage rights! I want that steel!

**Comment by MaaiG** [comment]
> I am not asking for a "complete" salvage of wrecked subs

At least a tiny one?

I had the desire to do this))
It may even be possible to create a small shelter in a wrecked submarine as a staging point on a route, but I doubt it’ll be able to return there.
By the way, it would be interesting to find a submarine of her own who died in previous sessions.
To find out what happened to your poor, fragile submarine that fell into the black abyss.
